opsmngr/organization_api_keys.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opsmngr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const apiKeysOrgPath = "api/public/v1.0/orgs/%s/apiKeys"
+
+// APIKeysServiceOp handles communication with the APIKey related methods
+// of the MongoDB Atlas API.
+type APIKeysServiceOp service
+
+var _ atlas.APIKeysService = &APIKeysServiceOp{}
+
+// APIKeysResponse is the response from the APIKeysService.List.
+type APIKeysResponse struct {
+	Links      []*atlas.Link  `json:"links,omitempty"`
+	Results    []atlas.APIKey `json:"results,omitempty"`
+	TotalCount int            `json:"totalCount,omitempty"`
+}
```

Comment:
any reason to redefine this? is it different from Atlas?

Reply:
in the atlas go client this is called `aPIKeysResponse` (private)